Sources/SmithyTimestamps/TimestampSerdeUtils.swift

```diff
@@ -84,174 +84,6 @@ public struct TimestampFormatter {
     }
 }
 
-// MARK: - Encoding Helpers
```

Comment:
Timestamps no longer need to be Codable so these extensions are removed.

TimestampFormatter (which converts Date to/from a string timestamp) remains, and is used in reading HTTP headers and by the JSON/XML/FormURL reader/writer classes.